students

```diff
@@ -1 +1,2 @@
 Ihor Kandaurov - https://github.com/lbrdev/Lesson1.LS.ld/
+Nastya Bezrukavaya - https://github.com/anAstyQ?tab=repositories

```

Comment:
@anAstyQ link should point to your concrete repository with task1.

Also check LICENSE file in your repo - it should contain valid apache or MIT license (check lecture for the licenses links)